src/org/usfirst/frc/team2706/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc.team2706.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Coordinates commands for the climber mechanism.
+ */
+public class Climber extends Subsystem {
+
+    private SpeedController motor = new Talon(6);
+    private boolean finished = false;
+
+    public void initDefaultCommand() {
+        // Set the default command for a subsystem here.
+        //setDefaultCommand(new MySpecialCommand());
+    }
+    
+    public void Climb() {
+        motor.set(0.5);
+    }
+    
+    public void Stop() {
```

Comment:
In Java, the naming convention for methods is camel case, similar to variables.
`stop()`